WNPRC_EHR/package-lock.json

```diff
@@ -5,27 +5,27 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.8.3",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.8.3.tgz",
-      "integrity": "sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==",
+      "version": "7.16.7",
```

Comment:
This does not appear to be generated with the correct version and/or command for what we expect from NPM. For best results I recommend the following:

1. Make sure you're using the same version of Node/NPM as is shipped with LabKey/Gradle. This is most easily done by adding a reference in your path. Something like:

```
export PATH="/Users/nickkerr/code/labkey/develop/.node/node/bin:/Users/nickkerr/code/labkey/develop/.node/npm/bin:$PATH"
```

Otherwise, you can install them natively. We use:

```
node=16.13.1
npm=8.1.2
```

2. To "rehydrate" the package-lock.json files I usually use the following command from the directory containing the package.json:

```
rm -rf node_modules && rm package-lock.json && npm install --legacy-peer-deps
```

3. You should have version "2" lockfiles after all said and done. See https://github.com/LabKey/wnprc-modules/blob/e6153c18/WNPRC_EHR/package-lock.json#L4 as an example from my PR.

Reply:
@dnicolalde Did you see this comment? I'm asking since you asked:

> Should I update my version of npm and node? and will it work on 21.3?